README.md

```diff
@@ -1 +1,41 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+
+> Description the project.
+
+
+## Built With
+
+- html
+- css
+
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle]([https://github.com/githubhandle](https://github.com/george827))
```

Comment:
- [ ] Kindly review that the link works as expected for a professional README file.